src/lib/Libattr/master_job_attr_def.xml

```diff
@@ -749,6 +749,29 @@
       </member_verify_function>
    </attributes>
    <attributes>
+      /* "JOB_ATR_Remove" */
+      <member_name>
+         <both>ATTR_R</both>
+      </member_name>
+      <member_at_decode>decode_str</member_at_decode>
+      <member_at_encode>encode_str</member_at_encode>
+      <member_at_set>set_str</member_at_set>
+      <member_at_comp>comp_str</member_at_comp>
+      <member_at_free>free_str</member_at_free>
+      <member_at_action>NULL_FUNC</member_at_action>
```

Comment:
Don't you need an action function here too? May be a function similar to verify_value_removefiles() which will help verify legit values when a hook is setting the value.